File: bazel_base/py2/bazel_base/bazel_base/bazel_base/SingleValue.py

import bazel_base.Value

class SingleValue(bazel_base.Value.Value):
    def stringify(self, spaces = ''):
    return '' if self.value is None else str(self.value)
    
        
    def __init__(self):
        super(SingleValue, self).__init__()
        self.value = ''
        pass
    

File: bazel_base/cpp/bazel_base/bazel_base/src/SingleValue.cpp

#include "SingleValue.h"

namespace bazel_base {

SingleValue::SingleValue() : SingleValue("") {}

SingleValue::~SingleValue() {}

void SingleValue::set_value(const string &value) { this->value_ = value; }

SingleValue::SingleValue(const string &value) : Value() {}

void SingleValue::stringify(ostream &out) const { cout << this->value_; }

} // namespace bazel_base
